CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 - Bug #98: Fix error on formatting trace, when it doesn't contain "file" and "line" (@vjik)
 - New #108: Support of nested values in message templates' variables, e. g. `{foo.bar}` (@vjik)
 - Bug #89: Fix error on parse messages, that contains variables that cannot cast to a string (@vjik)
+- New #109: Add context enricher (@vjik)
+- Chg #109: Deprecate `Logger` methods `setTraceLevel()` and `setExcludedTracePaths()` in favor of context enricher
```

Comment:
Should README be adjusted?

Reply:
Currently README don't contain information about this methods.

Reply:
Should it?

Reply:
Yes. Bun I think it should be in separated section "Context providers".

Reply:
Done